charts/kargo/templates/dex-server/deployment.yaml

```diff
@@ -74,25 +74,13 @@ spec:
         resources:
           {{- toYaml .Values.api.oidc.dex.resources | nindent 10 }}
         {{- if .Values.api.oidc.dex.probes.enabled }}
```

Comment:
Same as [this comment](https://github.com/akuity/kargo/pull/2906/files#r1832831666).